### ISSUE TITLE
Persistence of user between session

### ISSUE THREAD

**Issue by MicheleLonghi** [CloudBoost/JavaSDK#2]
After login or signup is essential that the user could stay logged in, even after the app restart.

**Comment by juicycleff** [comment]
Yes this is something they could have fixed a long time ago. For my team we had to implement it our self
